src/main/java/az/ingress/service/concurate/QueueServiceHandler.java

```diff
@@ -0,0 +1,17 @@
+package az.ingress.service.concurate;
+
+import az.ingress.queue.QueueSender;
+import az.ingress.queue.RatingQueueDto;
+import az.ingress.service.abstraction.QueueService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class QueueServiceHandler implements QueueService {
```

Comment:
it is unnecessary, you can use 
`private final QueueSender<RatingQueueDto> ratingQueueSender;`